doc_source/aws_construct_lib.md

```diff
@@ -1,111 +1,61 @@
 --------
 
-This documentation is for the developer preview release \(public beta\) of the AWS Cloud Development Kit \(CDK\)\. Releases might lack important features and might have future breaking changes\.
+This documentation is for the developer preview release \(public beta\) of the AWS Cloud Development Kit \(AWS CDK\)\. Releases might lack important features and might have future breaking changes\.
 
 --------
 
 # AWS Construct Library<a name="aws_construct_lib"></a>
```

Comment:
This is not the title of this topic. It should be "Maturity and Versioning", no?